Formula/grafana-agent.rb

```diff
@@ -31,12 +34,54 @@ def install
     system "go", "build", *std_go_args(ldflags: ldflags.join(" ")), "-o", bin/"grafana-agentctl", "./cmd/agentctl"
   end
 
+  def post_install
+    (etc/"grafana-agent").mkpath
+  end
+
+  def caveats
+    <<~EOS
+      The agent uses a configuration file that you must customize before running:
+        #{etc}/grafana-agent/config.yml
+    EOS
+  end
+
+  plist_options manual: "grafana-agent -config.file=#{HOMEBREW_PREFIX}/etc/grafana-agent/config.yml"
+
+  def plist
```

Comment:
The plist should be a `service do` block, and then you don't need the plist_options.

Reply:
Ahh excellent, thanks for the info! This looks like a much cleaner way to do this.